### ISSUE TITLE
Change default value to plus

### ISSUE THREAD

**Issue by aballano** [Shyish/CheckView#1]
By default, the selected value should be the plus, since the normal use flow is to add something and then show the check, not the other way around.

**Comment by deshant** [comment]
I agree, would have been better if default was plus instead of check.
